optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/test/java/org/optaplanner/quarkus/OptaPlannerProcessorOverridePropertiesAtRuntimeTest.java

```diff
@@ -77,6 +86,10 @@ public MyOptaPlannerRuntimeConfig() {
             solver.termination.bestScoreLimit = Optional.of("7");
             solver.termination.spentLimit = Optional.empty();
             solver.termination.unimprovedSpentLimit = Optional.empty();
+            solver.moveThreadCount = Optional.of("3");
+
+            solverManager = new SolverManagerRuntimeConfig();
+            solverManager.parallelSolverCount = Optional.of("10");
         }
```

Comment:
Is "MyOptaPlannerRuntimeConfig extends OptaPlannerRuntimeConfig" a good way to test this? I don't think anyone will ever extend that class - in fact, we might want to make OptaPlannerRuntimeConfig a final class to prevent users from doing that.

How does Quarkus extensions test a property that can be overwritten at runtime is capable of doing that?